docs/canonicalk8s/init.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
```

Comment:
I'm not sure if this file is necessary as the init for the starter pack has already been done for this repo